src/NzbDrone.Core/Parser/Parser.cs

```diff
@@ -619,7 +619,7 @@ private static ParsedTrackInfo ParseAudioTags(string path)
 
             var trackNumber = file.Tag.Track;
             var trackTitle = file.Tag.Title;
-            var discNumber = (file.Tag.Disc > 0) ? Convert.ToInt32(file.Tag.Disc) : 1;
+            var discNumber = (file.Tag.Disc > 0) ? Convert.ToInt32(file.Tag.Disc) : 0;
 
```

Comment:
Can I just change this to `var discNumber = file.Tag.Disc;`?  I think that's how the trackNumber will be working when that's missing

Reply:
I seem to remember there was an issue with number format. Which is why we did the int convert. I'd have to look back

Reply:
looks like its stored at uint from taglib so you should be able to just use file.Tag.Disc. You'll have to change DiscNumber to uint.